src/test/groovy/graphql/normalized/ExecutableNormalizedOperationFactoryTest.groovy

```diff
@@ -1314,7 +1314,7 @@ schema {
 
     private void assertValidQuery(GraphQLSchema graphQLSchema, String query, Map variables = [:]) {
         GraphQL graphQL = GraphQL.newGraphQL(graphQLSchema).build()
-        assert graphQL.execute(query, null, variables).errors.size() == 0
+        assert graphQL.execute(query, null, variables).errors.size() == 0 // Retain deprecated method for test coverage
```

Comment:
Move to its own home test  - remove from here - make a new test in its home test